packages/core/src/awsService/cloudWatchLogs/liveTailLogStreamSubmenu.ts

```diff
@@ -0,0 +1,168 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Prompter, PromptResult } from '../../shared/ui/prompter'
+import { DefaultCloudWatchLogsClient } from '../../shared/clients/cloudWatchLogsClient'
+import { createCommonButtons } from '../../shared/ui/buttons'
+import { createInputBox, InputBoxPrompter } from '../../shared/ui/inputPrompter'
+import { createQuickPick, DataQuickPickItem, QuickPickPrompter } from '../../shared/ui/pickerPrompter'
+import { pageableToCollection } from '../../shared/utilities/collectionUtils'
+import { CloudWatchLogs } from 'aws-sdk'
+import { isValidResponse, StepEstimator } from '../../shared/wizards/wizard'
+import { isNonNullable } from '../../shared/utilities/tsUtils'
+
+export enum LogStreamFilterType {
+    MENU = 'menu',
+    PREFIX = 'prefix',
+    SPECIFIC = 'specific',
+    ALL = 'all',
+}
+
+export interface LogStreamFilterResponse {
+    readonly filter?: string
+    readonly type: LogStreamFilterType
+}
+
+export class LogStreamFilterSubmenu extends Prompter<LogStreamFilterResponse> {
+    private logStreamPrefixRegEx = /^[^:*]*$/
+    private currentState: LogStreamFilterType = LogStreamFilterType.MENU
+    private steps?: [current: number, total: number]
+    private region: string
+    private logGroupArn: string
+    public defaultPrompter: QuickPickPrompter<LogStreamFilterType> = this.createMenuPrompter()
+
+    public constructor(logGroupArn: string, region: string) {
+        super()
+        this.region = region
+        this.logGroupArn = logGroupArn
+    }
+
+    public createMenuPrompter() {
+        const helpUri = 'https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_StartLiveTail.html'
+        const prompter = createQuickPick(this.menuOptions, {
+            title: 'Select LogStream filter type',
+            buttons: createCommonButtons(helpUri),
+        })
+        return prompter
+    }
+
+    private get menuOptions(): DataQuickPickItem<LogStreamFilterType>[] {
+        const options: DataQuickPickItem<LogStreamFilterType>[] = []
+        options.push({
+            label: 'All',
+            detail: 'Include log events from all LogStreams in the selected LogGroup',
+            data: LogStreamFilterType.ALL,
+        })
+        options.push({
+            label: 'Specific',
+            detail: 'Include log events from only a specific LogStream',
+            data: LogStreamFilterType.SPECIFIC,
+        })
+        options.push({
+            label: 'Prefix',
+            detail: 'Include log events from LogStreams that begin with a provided prefix',
+            data: LogStreamFilterType.PREFIX,
+        })
+        return options
+    }
+
+    public createLogStreamPrefixBox(): InputBoxPrompter {
+        const helpUri =
```

Comment:
same with this documentation link (just trying to point them out so you can click the resolve button on github when they're moved)